projects/legacy/styles/mixins/mixins.less

```diff
@@ -33,12 +33,6 @@
     appearance: none;
     word-break: keep-all;
     -webkit-text-fill-color: currentColor; // for Safari
-
-    .ios-only({
```

Comment:
Something like this exists in modern textfields too, please remove it there as well